client/src/components/page.jsx

```diff
@@ -27,7 +27,7 @@ class Page extends React.Component {
         const { userId } = songData;
         this.getUserData(userId);
       }).then(() => {
-        console.log('items included');
```

Comment:
Remove for production

Reply:
thanks for the catch! removed the .then () but kept the console log error on catch